crates/sui-graphql-e2e-tests/tests/stable/call/checkpoint_connection_pagination.exp

```diff
@@ -13,37 +13,12 @@ Response: {
         "hasPreviousPage": true,
         "hasNextPage": true
       },
-      "edges": [
-        {
-          "cursor": "eyJjIjoxMiwicyI6N30",
-          "node": {
-            "sequenceNumber": 7
-          }
-        },
-        {
-          "cursor": "eyJjIjoxMiwicyI6OH0",
-          "node": {
-            "sequenceNumber": 8
-          }
-        },
-        {
-          "cursor": "eyJjIjoxMiwicyI6OX0",
-          "node": {
-            "sequenceNumber": 9
-          }
-        },
-        {
-          "cursor": "eyJjIjoxMiwicyI6MTB9",
-          "node": {
-            "sequenceNumber": 10
-          }
-        }
-      ]
```

Comment:
Very confused as to why this changed. The cursor logic stayed the same, I only added the retrieval of raw checkpoints based on the sequence numbers that were fetched from the DBs...